### DISCUSSION TITLE
PnP Core Logging

### DISCUSSION THREAD

**Discussion by kentlhastings** [pnp/pnpcore#385] [Q&A] [answered]
Hello all,

Is there a handy way to disable Pnp Core logging messages appearing in the console? We're currently using a console app to handle our sharepoint operations, and our output window is filling up with status messages. My current workaround idea is to use a buffer and continuously clear the screen to only show our own messages, but perhaps there is a setting I can toggle?

Thanks!

**Comment by jansenbe** [comment, marked as answer]
You can configure logging: https://pnp.github.io/pnpcore/using-the-sdk/basics-settings.html. Setting to loglevel None should prevent output (https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1#log-level)

**Reply by kentlhastings** [reply]
Thank you for all of your help!
